uswds/uswds
# USWDS - HTMLProofer update broken links

## Summary
Remediate broken links found in updated HTMLProofer checks in USWDS-Site. 
I simply deleted links when I could not find an alternative location via search. If anyone has more familiarity with these showcase sites, please confirm that deleting these items is the best course. 

Related PR: https://github.com/uswds/uswds-site/pull/1735
Related issue: https://github.com/uswds/uswds-site/issues/1734

## Preview link
- [Who is using USWDS](https://github.com/uswds/uswds/blob/86dfb29d1f22c36693324ae2ea5a047be8ce2e24/docs/WHO_IS_USING_USWDS.md)
- [Security page](https://github.com/uswds/uswds/blob/86dfb29d1f22c36693324ae2ea5a047be8ce2e24/SECURITY.md)

## Testing and review
To review:
1. Confirm that updated urls work and point to the correct location.
2. Confirm that the deleted links need to be removed (there is no alternate location the link can point to).
3. Confirm that all items flagged in the USWDS-site `al-htmlproofer` branch are resolved.

## HTMLProofer error report
From the [`al-htmlproofer` branch](https://github.com/uswds/uswds-site/tree/al-htmlproofer) on the USWDS-Site repo. (Related USWDS-Site PR: https://github.com/uswds/uswds-site/pull/1735)

To reproduce: 
1. Open USWDS `al-htmlproofer` branch
2. Run `npm run proof` in USWDS-Site, receive the errors below.

```
For the Links > External check, the following failures were found:

* At ./_site/about/security/index.html:2507:

  External link https://designsystem.digital.gov/documentation/developers/#download-and-install failed: https://designsystem.digital.gov/documentation/developers/ exists, but the hash 'download-and-install' does not (status code 200)

* At ./_site/about/security/index.html:2507:

  External link https://designsystem.digital.gov/documentation/developers/#install-using-npm failed: https://designsystem.digital.gov/documentation/developers/ exists, but the hash 'install-using-npm' does not (status code 200)

* At ./_site/documentation/developers/index.html:2849:

  External link https://github.com/uswds/uswds/releases/download/v3.1.0/uswds-uswds-3.1.0.tgz failed (status code 404)

* At ./_site/getting-started/showcase/all/index.html:2612:

  External link http://gipsa.usda.gov/ failed with something very wrong.
It's possible libcurl couldn't connect to the server, or perhaps the request timed out.
Sometimes, making too many requests at once also breaks things. (status code 0)

* At ./_site/getting-started/showcase/all/index.html:2624:

  External link https://openforest.fs.usda.gov/christmas-trees/forests failed with something very wrong.
It's possible libcurl couldn't connect to the server, or perhaps the request timed out.
Sometimes, making too many requests at once also breaks things. (status code 0)

* At ./_site/getting-started/showcase/all/index.html:2647:

  External link https://epa-notice.usa.gov/ failed with something very wrong.
It's possible libcurl couldn't connect to the server, or perhaps the request timed out.
Sometimes, making too many requests at once also breaks things. (status code 0)

* At ./_site/getting-started/showcase/all/index.html:2648:

  External link https://fardatacollection.sam.gov failed with something very wrong.
It's possible libcurl couldn't connect to the server, or perhaps the request timed out.
Sometimes, making too many requests at once also breaks things. (status code 0)

* At ./_site/getting-started/showcase/all/index.html:2660:

  External link https://federalist-report-template.18f.gov/ failed with something very wrong.
It's possible libcurl couldn't connect to the server, or perhaps the request timed out.
Sometimes, making too many requests at once also breaks things. (status code 0)

* At ./_site/getting-started/showcase/all/index.html:2665:

  External link https://hud-digital-services.github.io/single-family-policy-handbook/ failed (status code 404)

* At ./_site/getting-started/showcase/all/index.html:2672:

  External link http://gsa.github.io/openIAE/ failed: Not Found (status code 404)

* At ./_site/getting-started/showcase/all/index.html:2682:

  External link https://jobkit.data.gov/ failed with something very wrong.
It's possible libcurl couldn't connect to the server, or perhaps the request timed out.
Sometimes, making too many requests at once also breaks things. (status code 0)

* At ./_site/getting-started/showcase/all/index.html:2689:

  External link https://meps.ahrq.gov/mepstrends/home/index.html failed (status code 401)

* At ./_site/getting-started/showcase/all/index.html:2710:

  External link https://omb-eregs-demo.app.cloud.gov/ failed (status code 404)

* At ./_site/getting-started/showcase/all/index.html:2727:

  External link https://match.state.gov/ failed with something very wrong.
It's possible libcurl couldn't connect to the server, or perhaps the request timed out.
Sometimes, making too many requests at once also breaks things. (status code 0)

* At ./_site/getting-started/showcase/all/index.html:2735:

  External link http://alexose.github.io/experiments/correspondence/ failed: Not Found (status code 404)

HTML-Proofer found 15 failures!
```
---

Before opening this PR, make sure you’ve done whichever of these applies to you:
